### PR TITLE
some changes to the default search page

### PR DESCRIPTION
**Pull request by ninjacazaam**

### CODE REVIEW

File: app/views/shared/_setlist_card.html.erb [hunk 1]
```diff
@@ -4,6 +4,8 @@
       <div class="imgBx">
         <% if setlist.setlist_image.attached? %>
           <%= cl_image_tag setlist.setlist_image.key %>
+        <% elsif setlist.tracks.empty? %>
+          <%= cl_image_tag 'app/assets/images/sets_logo.svg' %>
```
Comment (ninjacazaam) [inline app/views/shared/_setlist_card.html.erb:8]: This doesnt always work, @scallywag89 You may have to look over this one

Comment (scallywag89) [reply app/views/shared/_setlist_card.html.erb:8]: Will have a look, think I might see the issue there

Comment (scallywag89) [reply app/views/shared/_setlist_card.html.erb:8]: @ninjacazaam firstly, I think that we don't really need that elsif - if there is no user image i think it's cleaner to just use the first album cover. if we do want the elsif then it should be an image_tag not a cl_image_tag i'm pretty sure!

**Review by scallywag89 — CHANGES_REQUESTED** [review]

File: app/views/shared/_setlist_card.html.erb (hunk 1 above)
Comment (ninjacazaam) [reply app/views/shared/_setlist_card.html.erb:8]: If there is no user image and there are no tracks in the setlist then it throws an error, hence why this elsif exists

**Pull request merged**